src/data/set/basic.lean

```diff
@@ -131,22 +131,22 @@ theorem set.coe_eq_subtype (s : set α) : ↥s = {x // x ∈ s} := rfl
 @[simp] theorem set.coe_set_of (p : α → Prop) : ↥{x | p x} = {x // p x} := rfl
 
 @[simp] theorem set_coe.forall {s : set α} {p : s → Prop} :
-  (∀ x : s, p x) ↔ (∀ x (h : x ∈ s), p ⟨x, h⟩) :=
+  (∀ x : s, p x) ↔ ∀ x (h : x ∈ s), p ⟨x, h⟩ :=
 subtype.forall
 
 @[simp] theorem set_coe.exists {s : set α} {p : s → Prop} :
-  (∃ x : s, p x) ↔ (∃ x (h : x ∈ s), p ⟨x, h⟩) :=
+  (∃ x : s, p x) ↔ ∃ x (h : x ∈ s), p ⟨x, h⟩ :=
 subtype.exists
 
 theorem set_coe.exists' {s : set α} {p : Π x, x ∈ s → Prop} :
-  (∃ x (h : x ∈ s), p x h) ↔ (∃ x : s, p x x.2)  :=
+  (∃ x (h : x ∈ s), p x h) ↔ ∃ x : s, p x x.2 :=
 (@set_coe.exists _ _ $ λ x, p x.1 x.2).symm
 
 theorem set_coe.forall' {s : set α} {p : Π x, x ∈ s → Prop} :
-  (∀ x (h : x ∈ s), p x h) ↔ (∀ x : s, p x x.2)  :=
+  (∀ x (h : x ∈ s), p x h) ↔ ∀ x : s, p x x.2 :=
 (@set_coe.forall _ _ $ λ x, p x.1 x.2).symm
```

Comment:
I'm not at all convinced that this actually is an improvement; it breaks the symmetry.

Reply:
I disagree, but I also don't care all that much. I've reverted this.